### ISSUE TITLE
Kiwi Browser keeps disabling bypass paywalls clean

### ISSUE THREAD

**Issue by sbtristan98** [kiwibrowser/android#257]
Kiwi Browser often disables the bypass paywalls clean plugin on its own without me doing anything. How can I prevent this from happening?

**Comment by Asiaboy89** [comment]
I can confirm this behavior. After some weeks of being enabled, the addon gets disabled automatically. Doesn't matter if I clean the cache in the meantime or not.

Another user mentioned a similar behavior: https://www.reddit.com/r/firefox/comments/tig6lm/comment/i97uhze/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button

**Comment by sbtristan98** [comment]
It is telling me the newest version of "Bypass Paywalls Clean" has been disabled because it requires more permissions. Is there a way to enable the permissions by default for this particular extension?

**Comment by Asiaboy89** [comment]
Probably found a solution:
https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/issues/265

Means:
1) Remove your old BPC extension
2) Install "bypass-paywalls-chrome-clean-kiwi-custom.crx" at this site https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/releases
3) Activate the extension (don't worry, the version 3.2.0.0 will be updated to 3.2.3.0 automatically after Kiwi restart)
4) Enjoy

**Comment by kaoneko** [comment]
Yes, this happens when you use the regular version that only requests permissions for the sites it works on. If you don't mind, just install the version that requests access for all sites and it'll keep working after an update.
